tests/test_platform_telegram.py

```diff
@@ -0,0 +1,35 @@
+import aiohttp
+import pytest
+
+from bottery.platform.telegram.api import TelegramAPI
+from bottery.platform.telegram.engine import TelegramEngine
+
+
+def test_platform_telegram_api_non_existent_method():
+    api = TelegramAPI('token', aiohttp.ClientSession)
+    with pytest.raises(AttributeError):
+        api.non_existent_method()
+
+
+@pytest.mark.asyncio
+async def test_platform_telegram_api_get_updates():
+    api = TelegramAPI('token', aiohttp.ClientSession)
+    with pytest.raises(TypeError):
+        await api.get_updates()
```

Comment:
This one here should be a bit different. It's actually making a request to Telegram API. Would be better mock session and check if it was called.